site/viamguides/src/air-quality/air-quality.md

```diff
@@ -0,0 +1,340 @@
+1. Plug the box fan into the Kasa smart plug. 
+  ![photo of Kasa smart plug with fan](assets/smartPlug.jpg)
+
+<!-- ------------------------ -->
+## Set up your Raspberry Pi
+Duration: 6
+
+The Raspberry Pi boots from a USB flash drive (or microSD card). You need to install Raspberry Pi OS on a USB flash drive that you will use with your Pi. For more details about alternative methods of setting up your Raspberry Pi, refer to the [Viam docs](https://docs.viam.com/installation/prepare/rpi-setup/#install-raspberry-pi-os).
+
+### Install Raspberry Pi OS
+1. Connect the USB flash drive (or microSD card) to your computer.
+1. Download the [Raspberry Pi Imager](https://www.raspberrypi.com/software/) and launch it.
+  ![raspberry pi imager](assets/imager.png)
+1. Click **CHOOSE DEVICE**. Select your model of Pi, which is Raspberry Pi 5.
+1. Click **CHOOSE OS**. Select **Raspberry Pi OS (64-bit)** from the menu.
+1. Click **CHOOSE STORAGE**. From the list of devices, select the USB flash drive you intend to use in your Raspberry Pi.
+  ![raspberry pi storage](assets/osSelect.png)
+1. Configure your Raspberry Pi for remote access. Click **Next**. When prompted to apply OS customization settings, select **EDIT SETTINGS**.
+1. Check **Set hostname** and enter the name you would like to access the Pi by in that field, for example, `air`.
+1. Select the checkbox for **Set username and password** and set a username (for example, your first name) that you will use to log into the Pi. If you skip this step, the default username will be `pi` (not recommended for security reasons). And specify a password.
+1. Connect your Pi to Wi-Fi so that you can run `viam-server` wirelessly. Check **Configure wireless LAN** and enter your wireless network credentials. SSID (short for Service Set Identifier) is your Wi-Fi network name, and password is the network password. Change the section `Wireless LAN country` to where your router is currently being operated.
+  ![raspberry pi hostname username and password](assets/generalSettings.png)
+1. Select the **SERVICES** tab, check **Enable SSH**, and select **Use password authentication**.
+    ![raspberry pi enable SSH](assets/sshSettings.png)
+    > aside negative
+    > Be sure that you remember the `hostname` and `username` you set, as you will need this when you SSH into your Pi.
+1. **Save** your updates, and confirm `YES` to apply OS customization settings. Confirm `YES` to erase data on the USB flash drive. You may also be prompted by your operating system to enter an administrator password. After granting permissions to the Imager, it will begin writing and then verifying the Linux installation to the USB flash drive.
+1. Remove the USB flash drive from your computer when the installation is complete.
+
+### Connect with SSH
+
+1. Place the USB flash drive into your Raspberry Pi and boot the Pi by plugging it in to an outlet. A red LED will turn on to indicate that the Pi is connected to power.
```

Comment:
I like the idea of using a USB flash drive by default over an SD card for longevity and availability (most folks probably have one or two lying around). We should be sure to call out anything that could prevent booting from USB working out of the box, like using a lower amperage power supply with the Raspberry Pi 5:

> USB boot is disabled by default when connected to a 3A power supply. 
https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#differences-on-raspberry-pi-5

The power supply that should be included by Raspberry Pi is 5V 5A (25W). However, folks who have owned a Pi before or got an unofficial power supply might be using 5V 3A (15W) since it was the default previously. 

I know you called out the need for 5V in a previous step, but we should probably include the 5A (amps) requirement for optimal performance too.